quickstart corrections (#28)
terramate-io/terramate-docs: cli/getting-started/index.md

```
:::

Other installation options [are available](../installation.md#alternative-installation-methods). When the installation is complete, you can test it out by reading the current version:

```sh
$ terramate version
```

### Install Terraform or OpenTofu

> Next, install Terraform or OpenTofu to orchestrate it using Terramate:

Reviewer: ```suggestion
Next, install Terraform or OpenTofu:
```

sounds weird, maybe we just ditch orchestration here?

Author: yes, I think it's not needed here.

- [Install Terraform](https://developer.hashicorp.com/terraform/install)
- [Install OpenTofu](https://opentofu.org/docs/intro/install/)

## Create a new project

Terramate requires a git repository, and every git repository is a project in Terramate.

> Let's create a new directory called `terramate-quickstart` and initialize Git to set up your first Terramate project.

Reviewer: ```suggestion
Let's create a new repository `terramate-quickstart` to set up your first Terramate project.
```

- How to create a [Terramate Cloud](https://cloud.terramate.io) account and connect it to your project.

Those examples hopefully give you a starting point and help you get insights into the capabilities of Terramate.

> This quickstart is just the tip of the iceberg. Running code generation and change detection on the scale can help you save a lot of time when maintaining or running Terraform.

Reviewer: ```suggestion
This quickstart is just the tip of the iceberg. Running code generation and change detection can help you save a lot of time when maintaining or running Terraform.
```
